### DISCUSSION TITLE
Insufficient privileges to get access token

### DISCUSSION THREAD

**Discussion by ObviousViking** [SteamRE/DepotDownloader#379] [Q&A]
Now i think im right in thinking that this is just the way that the script should work.

However its worth an ask.

I get the error 'Insufficient privileges to get access token' when attempting to download some depots. I assume as my account which logs in successfully doesn't own the app its trying to access.

However, i did use -manifest-only so surely it should let me access a list of the files rather than the files themsevles?

**Comment by psychonic** [comment]
In addition to the file content, even the file name are encrypted in the manifest and require the depot key to be decrypted. You cannot get that without owning the app.

Other file info is technically available (size, flags), but doesn't seem like it's worth the effort to expose.
